actionmailbox/app/models/action_mailbox/inbound_email/message_id.rb

```diff
@@ -10,7 +10,7 @@ module ActionMailbox::InboundEmail::MessageId
   extend ActiveSupport::Concern
 
   class_methods do
-    # Create a new +InboundEmail+ from the raw +source+ of the email, which be uploaded as a Active Storage
+    # Create a new +InboundEmail+ from the raw +source+ of the email, which be uploaded as an Active Storage
```

Comment:
> which be uploaded

Could you fix the grammar here too?

Reply:
Is this grammar ok?
`which be uploaded` -> `which is uploaded`